### DISCUSSION TITLE
通过tryconfigs给ndk-build传递参数

### DISCUSSION THREAD

**Discussion by caxiejiang** [xmake-io/xmake#3012] [Q&A]
目前我通过xmake f -p android trybuild=ndkbuild命令可以使用xmake编译Android.mk脚本，同时我还想向Android.mk脚本里面传递参数。我看到文档中提供了tryconfigs可以向构建系统中传命令，但是看起来tryconfigs并没有起作用，我使用的命令如下：
xmake f -p android --ndk=%NDK_ROOT% --trybuild=ndkbuild --tryconfigs="--MODULENAME=xxxx"

**Comment by waruqi** [comment]
目前仅对 autoconf/cmake 支持传参。。

**Reply by caxiejiang** [reply]
了解，谢谢您的解答
